scout/adapter/mongo/omics_variant.py

```diff
@@ -82,8 +82,8 @@ def set_samples(self, case_obj: dict, omics_model: dict):
         )
 
         sample = {
-            "sample_id": match.get("omics_sample_id", match["individual_id"]),
-            "display_name": match["display_name"],
+            "sample_id": match["individual_id"],
```

Comment:
The variant query is using individual_id to collect all variants to display on the WTS variants page, so this has to be like this.

Reply:
Did it end up on the wrong line after all those changes? 🙄Well spotted. But loading a case on stage worked just fine?! Good with many fallbacks I guess. I'll test through a few other scenarios.

Reply:
Loading a case on stage did work, but then no vars were shown

Reply:
Im guessing your case had a different setup, and either hit a different fallback?

Reply:
Yes, probably!

Reply:
But it's replicable also with the demo instance, try

Reply:
Quite, but there are apparently more moving parts here.